packages/ListBox/src/components/Filter/Filter.js

```diff
@@ -67,30 +61,38 @@ const Filter = React.forwardRef((props, ref) => {
     applyFilter(dispatch, applyFilterType)(filteredOptions, noResultsFound);
   };
 
-  const handleBlur = () => {
-    window.requestAnimationFrame(() => {
-      if (hasFooter) {
-        // if has footer the responsible of closing is the Footer no the onblur event
-        return;
-      }
-
-      if (document.activeElement !== refListBoxContainer.current) {
-        // this will reset the activeOption and close the Popover
+  React.useEffect(() => {
+    let id = null;
+    if (!isOpen) {
+      id = window.requestAnimationFrame(() => {
+        applyFilter(dispatch, applyFilterType)([], false);
+        setTextSearch("");
+      });
+
+      return () => {
+        cancelAnimationFrame(id);
+      };
+    }
+  }, [applyFilterType, dispatch, isOpen]);
+
+  React.useLayoutEffect(() => {
+    /**
+     * This prevents from scrolling on the input search and moving the popover into a wrong absolute position
+     */
+    function handleScroll() {
+      if (document.activeElement.dataset?.pkaAnchor === "list-filter-input") {
```

Comment:
Interesting 🤔 
But if the focus is _not_ on the `list-filter-input`, the `<Popover>` still behaves incorrectly.  It seems more likely to be a `<Popover>` bug though, and I'm not sure we should try to fix it here in the `<ListBox>`, or even in this PR.

Reply:
I removed this one, now seems stable don't know why 🤷